tests/reference_emails/done-failing.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>[ODA][done] failing first requested at 2021-08-19 17:25:52 job_id: f178fec5</title>
+</head>
+
+<body>
+Dear User,<br>
+<br>
+you receive this message because at 2021-08-19 17:25:52 ( 0.0 seconds ago ) you submitted a request <br>
+for a failing from the service provided by University of Geneva<br>
+available at the URL PRODUCTS_URL.<br>
+<br>
+
+Your request is now <b>ready</b>.<br>
+
+The following additional information have been returned regarding your request:<br>
+failing query
+Instrument: empty, product: failing failed!
```

Comment:
That's just beatiful, reminds of the classic:

![image](https://user-images.githubusercontent.com/3909535/176696379-5bb3219e-b829-45f5-a048-8a08933c70a2.png)

has to be adapted

Reply:
It's kind of a funny coincidence:

The error message is built here:

https://github.com/oda-hub/dispatcher-app/blob/master/cdci_data_analysis/analysis/instrument.py#L340-L358

This is then included in the `error_message` field of the `RemoteException`